clerk.go

```diff
@@ -169,6 +170,7 @@ func (h *CustomRequestHeaders) apply(req *http.Request) {
 		return
 	}
 	req.Header.Add("X-Clerk-Application", h.Application)
+	req.Header.Add("True-Client-IP", h.TrueClientIP)
```

Comment:
🔧 This is a semi-standard header name and can potentially cause issues if it's being set on the request directly. Better use the `X-Clerk-` namespace here as well?

Reply:
Done! That makes sense!